### PR TITLE
Update UG documentation

### PR DESCRIPTION
**Pull request by Bubbl3T**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -41,7 +37,7 @@ If you encounter any issues in launching and using the app, feel free to refer t
 * Items in square brackets are optional.
   * e.g. `n/NAME [c/CLASS_NAME]` can be used as `n/John Doe c/CS1231S` or as `n/John Doe`.
 * Items with `...` after them can be used multiple times.
-  * e.g. `c/CLASS_NAME...` can be used as `c/CS1101S`, `c/CS2030 c/ST2334`, etc.
+  * e.g. `c/CLASS_NAME...` can be used as `c/CS1101S`, or `c/CS2030 c/ST2334`.
```
Comment (Berted) [inline docs/UserGuide.md]:
```suggestion
  * e.g. `c/CLASS_NAME...` can be used as `c/CS1101S` or `c/CS2030 c/ST2334`.
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -55,9 +51,7 @@ If you encounter any issues in launching and using the app, feel free to refer t
 </div>
 
 ## Modes
-In TA-Assist, you can switch into a mode called the **focus** mode, which lets you run tasks that are specific to a (module) class. Therefore,
-[some commands](#features-available-in-focus-mode) can only be executed when you are in focus mode. Commands that you can run in the default (unfocused) mode can also
-be run in focus mode. On the other hand, commands that are available only in focus mode cannot be executed in the default (unfocused) mode.
+In TA-Assist, you can switch into a mode called the **focus** mode, which lets you run tasks that are specific to a class. Therefore, [some commands](#focus-mode-commands) can only be executed when you are in focus mode. Commands that you can run in the default (unfocused) mode can also be run in focus mode. On the other hand, commands that are available only in focus mode cannot be executed in the default mode.
```
Comment (Berted) [inline docs/UserGuide.md]:
```suggestion
In TA-Assist, you can switch into a mode called **focus** mode, which lets you run tasks that are specific to a class. Therefore, [some commands](#focus-mode-commands) can only be executed when you are in focus mode. Commands that you can run in the default (unfocused) mode can also be run in focus mode. On the other hand, commands that are available only in focus mode cannot be executed in the default mode.
```

File: docs/UserGuide.md [hunk 3]
```diff
@@ -3,12 +3,9 @@ layout: page
...
 ---
 
-**TA-Assist** is a desktop app for Teaching Assistants (TA) to keep track of students’ particulars and allocate marks 
-for attendance and assignments. It is optimized for use via a Command Line Interface (CLI) while having the benefits of 
-a Graphical User Interface (GUI).
+**TA-Assist** is a desktop app for Teaching Assistants (TA) to keep track of students’ particulars and allocate marks for attendance and assignments. It is optimized for use via a Command Line Interface (CLI) while having the benefits of a Graphical User Interface (GUI).
 
-This user guide provides brief documentation on how you can install the application and describes how each feature 
-should be used. Start by looking at the [quick start](#quick-start) guide to get you started.
+This user guide provides brief documentation on how you can install the application and describes how each feature should be used. Start by looking at the [quick start](#quick-start) guide to get you started.
```
Comment (njxue) [inline docs/UserGuide.md]:
```suggestion
This User Guide provides brief documentation on how you can install the application and describes how each feature should be used. Start by looking at the [quick start](#quick-start) guide to get you started.
```

**Review by Berted — CHANGES_REQUESTED** [review]
Just some minor grammar nits.

**Review by njxue — CHANGES_REQUESTED** [review]

**Review by njxue — APPROVED** [review]

File: docs/index.md [hunk 4]
```diff
@@ -8,12 +8,12 @@ title: TA-Assist
 
 ![Ui](images/Ui.png)
 
-**TA-Assist is a desktop application for managing your students' grades.** While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
+**TA-Assist is a desktop application for managing your students' grades.** While it has a GUI (Graphical User Interface), most of the user interactions happen from its CLI (Command Line Interface).
```
Comment (Berted) [inline docs/index.md]:
```suggestion
**TA-Assist is a desktop application for managing your students' grades.** While it has a GUI (Graphical User Interface), most of the user interactions are performed through a CLI (Command Line Interface).
```

**Review by Berted — APPROVED** [review]
Nice work, I have a minor nit regarding the grammar again however 🙏

File: docs/index.md (hunk 4 above)
Comment (Bubbl3T) [reply docs/index.md]: Alright sure!

**Pull request merged**